framework/src/bundle/BundleStorage.h

```diff
@@ -39,27 +40,22 @@ struct BundleArchive;
 struct BundleStorage
 {
 
-  virtual ~BundleStorage() {}
+  BundleStorage() {}
 
-  /**
-   * Insert bundle library into persistent storagedata.
-   *
-   * @param location Location of bundle to install.
-   * @return A list of BundleArchive instances representing the installed bundles.
-   */
-  virtual std::vector<std::shared_ptr<BundleArchive>> InsertBundleLib(
-    const std::string& location) = 0;
+  virtual ~BundleStorage() {}
 
   /**
    * Insert bundles from a container into persistent storagedata.
    *
    * @param resCont The container for the bundle data and resources.
-   * @param topLevelEntries The top level entries in the container to be inserted as bundle archives.
-   * @return A list of BundleArchive instances representing the installed bundles.
+   * @param topLevelEntry The top level entries in the container to be inserted as bundle archives.
```

Comment:
Is there a reason why this became singular? If so, should the comment also reflect the change?

Reply:
Yes. The call used to be "CreateAndInsertArchives" and a vector was passed in for that parameter. The looping was pulled out of this function and moved to the caller. There were 2 different code paths: 1 for a single archive creation, and 1 for multiples... the code was simplified so that in all code paths, only the singular case is done.